pyglet/window/xlib/__init__.py

```diff
@@ -232,6 +232,9 @@ def _create(self):
             root = xlib.XRootWindow(self._x_display, self._x_screen_id)
 
             visual_info = self.config.get_visual_info()
+            depth = 32 if self.style == "transparent" else 24
```

Comment:
I think we need to add the new `overlay` here as well. Something like: 
`depth = 32 if self.style in ('transparent', 'overlay') else 24`